zh_tw/news/_posts/2017-09-14-webrick-basic-auth-escape-sequence-injection-cve-2017-10784.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: news_post
+title: "CVE-2017-10784: WEBrick 基本認證的轉義序列注入安全性風險"
+author: "usa"
+translator: "Vincent Lin"
+date: 2017-09-14 12:00:00 +0000
+tags: security
+lang: zh_tw
+---
+
+Ruby 內建的 WEBrick 基本認證有轉義序列注入的安全性風險。
+此風險的 CVE 識別號已被指派為 [CVE-2017-10784](http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-10784) 。
+
+## 詳細
+
+當使用 WEBrick 的基本認證時，客戶端可以傳入任意字串作為使用者名稱。
+WEBrick 完整地將使用者名稱紀錄至紀錄檔（log）中，而攻擊者可通過紀錄檔注入惡意的轉義序列進而操縱受害者在終端機（terminal）執行危險的控制字符。
+
+此安全性風險與[另一個已修復的風險](/en/news/2010/01/10/webrick-escape-sequence-injection/)相似，但在基本認證中並沒有修復。
```

Comment:
`/en/news/` do we have a chinese translation?

Reply:
this post only have en/ja/ko version 😭

Reply:
We can translate in future PRs :)